### PR TITLE
Graphite tags parser

### PR DESCRIPTION
**Pull request by hitalos**
Fixing #4372 

### Required for all PRs:

- [X] Signed [CLA](https://influxdata.com/community/cla/).
- [X] Associated README.md updated.
- [X] Has appropriate unit tests.

### CODE REVIEW

File: plugins/parsers/graphite/parser.go [hunk 1]
```diff
@@ -147,6 +149,24 @@ func (p *GraphiteParser) ParseLine(line string) (telegraf.Metric, error) {
...
 	}
+
+	// Split name and tags
+	if len(parts) >= 2 {
+		for _, tag := range parts[1:] {
+			tagValue := strings.Split(tag, "=")
+			if len(tagValue) != 2 || len(tagValue[0]) == 0 || len(tagValue[1]) == 0 {
+				continue
+			}
+			if strings.IndexAny(tagValue[0], "!^") != -1 {
```
Comment (ssoroka) [inline plugins/parsers/graphite/parser.go:160]: fyi strings.ContainsAny

**Review by ssoroka — APPROVED** [review]

**Comment by endersonmaia** [comment]
@ssoroka hi, how to know in which version this will be released ?

**Comment by endersonmaia** [comment]
> @ssoroka hi, how to know in which version this will be released ?

https://github.com/influxdata/telegraf/releases/tag/v1.17.0

**Comment by ssoroka** [comment]
we'll get it into 1.17.1 for you. Should be in a couple weeks. You can use the nightly build to get it until then

**Comment by endersonmaia** [comment]
I just tested the `telegraf:1.17` docker image and it's working as expected :)

**Pull request merged**